File: phm-ui/src/model/entiy/MedicalOrder.ts
// 类型定义
export interface MedicalOrder {
  id?: number | null;       // 订单ID（新增时可选）
  itemId: number | null;    // 医疗项目ID（必填）
  petId: number | null;     // 宠物ID（必填）
  userId: number | null;    // 用户ID（必填）
}

File: phm-server/src/main/java/com/phm/model/entity/MedicalOrder.java
package com.phm.model.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

import java.util.Date;

@Data
@TableName("medical_order")
public class MedicalOrder {
    @TableId(type = IdType.AUTO)
    private Long id;          // 订单主键ID
    private Long itemId;      // 关联医疗项目ID（对应medical_item.id）
    private Long petId;       // 关联宠物ID（对应pet.pet_id）
    private Long userId;      // 关联用户ID（对应client.client_id）
    private Date createdAt;   // 创建时间（自动生成）
}
